src/Analysis/Ast/Impl/Specializations/Typing/TypingModule.cs

```diff
@@ -227,27 +235,32 @@ private IPythonType CreateTupleType(IReadOnlyList<IPythonType> typeArgs)
             return Interpreter.UnknownType;
         }
 
-        private IPythonType CreateTypeAlias(IArgumentSet args) {
-            var typeArgs = args.Values<IMember>();
-            if (typeArgs.Count == 2) {
-                var typeName = (typeArgs[0] as IPythonConstant)?.Value as string;
-                if (!string.IsNullOrEmpty(typeName)) {
-                    return new TypeAlias(typeName, typeArgs[1].GetPythonType() ?? Interpreter.UnknownType);
-                }
+        private IPythonType CreateTypeAlias(IArgumentSet argSet) {
+            if (argSet.Errors.Count > 0) {
+                argSet.ReportErrors();
+                return Interpreter.UnknownType;
+            }
+
+            var args = argSet.Values<IMember>();
+            var name = (args[0] as IPythonConstant)?.GetString();
```

Comment:
need to `check args.Count > 0`? Or never happens?

Reply:
It will never happen, see below.